kpatch-build/kpatch-cc

```diff
@@ -13,7 +13,7 @@ fi
 
 declare -a args=("$@")
 
-if [[ "$TOOLCHAINCMD" =~ "gcc" ]] ; then
+if [[ "$TOOLCHAINCMD" =~ "gcc" || "$TOOLCHAINCMD" =~ "clang" ]] ; then
```

Comment:
can we do this matching instead?

[[ "$TOOLCHAINCMD" =~ .*gcc$ || "$TOOLCHAINCMD" =~ .*clang$ ]]

name of executable binary for gcc or clang can have its own prefix to represent cpu architecture and platform. e.g., x86_64-linux-gnu-clang

Reply:
Because it uses regex matching right now it is even more relaxed than what you propose, i.e. it matches these substrings anywhere in TOOLCHAINCMD. But I think it is a good idea to make sure they are  at the end of the string.